src/Entity/EventCategory.php

```diff
@@ -22,4 +22,19 @@
  */
 class EventCategory extends BaseEventCategory
 {
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\EventGroupCategory", inversedBy="eventCategories")
+     * @ORM\JoinColumn(name="event_group_category_id", referencedColumnName="id", nullable=false)
+     */
+    private $eventGroupCategory;
+
+    public function getEventGroupCategory(): ?EventGroupCategory
```

Comment:
why `?`?

Reply:
form the form the groupCategory is not preset